docs/components/input/use-cases.md

```diff
@@ -13,6 +13,17 @@ Can be provided via the `label` attribute or via the `slot="label"`.
 export const label = () => html` <lion-input label="Input" name="input"></lion-input> `;
 ```
 
+## Label sr-only (a11y)
+
+Can be provided via the `label-sr-only` boolean attribute.
+
+The label will be hidden, but still readable by screen readers.
```

Comment:
It might be worth adding here that as soon as we support ElementInternals api, the equivalent would be `<lion-input aria-label="Input" name="input"></lion-input>`